content/outcomes/application-development-how-we-work/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How Tanzu Labs App Services Works and Why
+weight: 1
+layout: single
+description: "Learn our philosophy to enabling amazing outcomes through software."
```

Comment:
This doesn't sound quite right to me. Do you enable an outcome, or do you enable people to achieve an outcome?

Reply:
🤷‍♂️  This is why we're seriously reconsidering outcomes as a metaphor.

Reply:
We'll keep iterating on it.

Reply:
Ah, I see what you mean now 😉. Updated.